charts/ingress-nginx/values.yaml

```diff
@@ -525,6 +525,8 @@ controller:
       # allocateLoadBalancerNodePorts: true
       # -- Used by cloud providers to connect the resulting internal LoadBalancer to a pre-existing static IP. Make sure to add to the service the needed annotation to specify the subnet which the static IP belongs to. For instance, `networking.gke.io/internal-load-balancer-subnet` for GCP and `service.beta.kubernetes.io/aws-load-balancer-subnets` for AWS.
       loadBalancerIP: ""
+      # -- Type of the internal controller service.
+      type: LoadBalancer
```

Comment:
Please leave this empty so it's defaulting to `controller.service.type` as it did before.

Imagine someone set `controller.service.type: ClusterIP` in their current config. With the old behavior, they would get an internal service with `type: ClusterIP`.

With `controller.service.internal.type` being set to `LoadBalancer` by default, the `default` expression in the Helm template results in `LoadBalancer` for the internal service and therefore changes existing resources.

Reply:
I think that's also the reason why the tests are failing atm.